.github/workflows/nightlies.yml

```diff
@@ -0,0 +1,57 @@
+name: Upload nightly wheels to Anaconda Cloud
+
+on:
+  # Run daily at 1:23 UTC to upload nightly wheels to Anaconda Cloud
+  schedule:
+  - cron: '23 1 * * *'
```

Comment:
This can get changed to be anything that people would like. Chosen just to miss the `00:00` and `00:01` nightly rush.